lib/puppet/provider/cisco_overlay_global/nxapi.rb

```diff
@@ -41,60 +41,44 @@
     :dup_host_mac_detection_timeout,
   ]
 
-  PuppetX::Cisco::AutoGen.mk_puppet_methods(:non_bool, self, '@vxlan_global',
-                                            VXLAN_GLOBAL_PROPS)
+  PuppetX::Cisco::AutoGen.mk_puppet_methods(:non_bool, self, '@overlay_global',
```

Comment:
Just an fyi - Mike and I have been using `@nu` for this var name on the new properties. It makes it a little more clear which object it's referring to and also allows us to cut & paste the code with fewer changes.